backend/django_core/templates/anime/home.html

```diff
@@ -196,7 +196,6 @@
 {% endblock %}
 
 {%block js %}
-    <script src="https://cdn.jsdelivr.net/npm/swiper@11/swiper-bundle.min.js"></script>
     {% component_js_dependencies %}
     <script>
         const swiper = new Swiper('.swiper', {
```

Comment:
Add anonymous functions here. Read more about [this](https://baseplate-admin.github.io/blog/how-i-fixed-htmx-redeclaration-of-let-element-with-hx-boosttrue.html)